Travis builds broken

## Steps to Reproduce the Problem

Issue a CI build.

## Expected Result

Success for #3462

## Actual Result

```
$ cmake ${CMAKE_OPT[@]}
901CMake Error at /usr/local/Cellar/cmake/3.17.2/share/cmake/Modules/CMakeDetermineCCompiler.cmake:49 (message):
902  Could not find compiler set in environment variable CC:
903
904  gcc-9.
905Call Stack (most recent call first):
906  CMakeLists.txt:23 (project)
```

https://travis-ci.org/github/ElektraInitiative/libelektra/jobs/712141684

## System Information

- Elektra Version: master

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -28,7 +28,7 @@ bsd_task:
 
     - name: 😈 FreeBSD 11
       freebsd_instance:
-        image_family: freebsd-11-3-snap
+        image_family: freebsd-11-4
 
     - name: 😈 FreeBSD 12
       freebsd_instance:

--- a/.travis.yml
+++ b/.travis.yml
@@ -211,8 +211,8 @@ matrix:
 before_install:
   - |
     if [ "$TRAVIS_OS_NAME" = 'osx' ]; then
-      rvm install 2.6.5
-      rvm use 2.6.5
+      rvm install 2.6.6
+      rvm use 2.6.6
       gem install test-unit --no-document
       if [ "$CC" = 'gcc' ]; then
         brew upgrade gcc@9

--- a/.travis.yml
+++ b/.travis.yml
@@ -215,9 +215,9 @@ before_install:
       rvm use 2.6.6
       gem install test-unit --no-document
       if [ "$CC" = 'gcc' ]; then
-        brew upgrade gcc@9
-        export CC=gcc-9
-        export CXX=g++-9
+        brew upgrade gcc
+        export CC=gcc-10
+        export CXX=g++-10
       fi
       pip2 install cheetah # Required by kdb-gen
       # start D-Bus session bus

--- a/doc/news/_preparation_next_release.md
+++ b/doc/news/_preparation_next_release.md
@@ -157,7 +157,7 @@ you up to date with the multi-language support provided by Elektra.
 
 ### Cirrus
 
-- <<TODO>>
+- Update FreeBSD images from version 11.3 to 11.4. _(Mihael Pranjić)_
 - <<TODO>>
 - <<TODO>>
 
@@ -169,7 +169,7 @@ you up to date with the multi-language support provided by Elektra.
 
 ### Travis
 
-- <<TODO>>
+- Update macOS builds to use gcc 10 and ruby 2.6.6. _(Mihael Pranjić)_
 - <<TODO>>
 - <<TODO>>
 

